### PR TITLE
clean notebook IDs

### PR DESCRIPTION
**Pull request by CagtayFabry**
## Changes
adds a small function `util._clean_notebook_ids` to remove the autogenerated JuypterLab `ID` field from notebooks for better diffs when needed.

### CODE REVIEW

**Comment by vhirtham** [comment]
Maybe we should put this into the scripts folder and add a script that automatically clears all our notebooks

**Comment by CagtayFabry** [comment]
> 
> 
> Maybe we should put this into the scripts folder and add a script that automatically clears all our notebooks

good idea :)

**Review by marscher — APPROVED** [review]
LGTM

File: scripts/clean_notebooks.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# clean up jupyter notebooks in main directory
+if __name__ == "__main__":
+    from pathlib import Path
+
+    from weldx.util import _clean_notebook_ids
+
+    p = Path(__file__).parents[1]
+    print(p)
```
Comment (vhirtham) [inline scripts/clean_notebooks.py:8]: Supposed to stay? 😛

**Review by vhirtham — APPROVED** [review]
Maybe we can also execute

~~~
jupyter nbconvert --clear-output --inplace NOTEBOOKNAME.ipynb
~~~

per notebook to clear the cells too

File: scripts/clean_notebooks.py (hunk 1 above)
Comment (vhirtham) [inline scripts/clean_notebooks.py:8]:
```suggestion
```

**Comment by CagtayFabry** [comment]
> 
> 
> Maybe we can also execute
> 
> ```
> jupyter nbconvert --clear-output --inplace NOTEBOOKNAME.ipynb
> ```
> 
> per notebook to clear the cells too

I like it, I removed the outputs (and execution_count) manually so that there would be no need for nbconvert as a dependency

**Review by vhirtham — APPROVED** [review]
Really nice. Maybe I will set it up as a local pre-commit hook. Will definitely save some time.

**Comment by marscher** [comment]
nbdime will be your friend in terms of pre-commit hooks (and notebook diffs etc.)

**Pull request merged**